cypress/e2e/tests/pages/user_menu/preferences.spec.ts

```diff
@@ -0,0 +1,368 @@
+    */
+    const themeOptions = {
+      Light: 'theme-light',
+      Dark:  'theme-dark',
+      Auto:  ''
+    };
+    // Set theme for 'Auto' mode based on time of day
+    const hour = new Date().getHours();
+
+    for (const key in themeOptions) {
+      (key === 'Auto' && hour < 7 || hour >= 18) ? themeOptions['Auto'] = 'theme-dark' : themeOptions['Auto'] = 'theme-light';
+    }
+
+    PagePo.goTo(endpoint);
+    for (const [key, value] of Object.entries(themeOptions)) {
+      PagePo.goTo(endpoint);
+      prefPage.button(key).set();
+      prefPage.button(key).isSelected();
+      prefPage.checkThemeDomElement(value);
+      userMenu.open();
+      userMenu.checkOpen();
+      userMenu.clickMenuLink('Log Out');
+      cy.login();
+      PagePo.goTo(endpoint);
+      prefPage.button(key).isSelected();
+      prefPage.checkThemeDomElement(value);
+    }
+  });
+
+  it('Can select login landing page', () => {
+    /*
+    Select each radio button and verify its highlighted
+    Verify user is landing on correct page after login
+    Verify selection is preserved after logout/login
+    */
+    const landingPageOptions: {[key: number]: string} = {
+      0: '/home',
+      1: 'c/_/manager/provisioning.cattle.io.cluster',
+      // 2: '/explore' // TODO this option only works when there is an existing cluster (not for standard user)
+    };
+
+    for (const [key, value] of Object.entries(landingPageOptions)) {
+      PagePo.goTo(endpoint);
+      prefPage.radioButton().set(key);
+      prefPage.radioButton().isChecked(key);
+
+      // if key is 1, navigate to cluster manager page and then do validations, else just do validations
+      if (key == 1) {
+        cy.intercept('GET', '/v3/clusters').as('clusters');
+        PagePo.goTo('c/_/manager/provisioning.cattle.io.cluster');
+        cy.wait('@clusters').its('response.statusCode').should('eq', 200);
+      }
+
+      userMenu.open();
+      userMenu.checkOpen();
+      userMenu.clickMenuLink('Log Out');
+      cy.login();
+      cy.visit(Cypress.config().baseUrl);
+      cy.url().should('include', value);
+      PagePo.goTo(endpoint);
+      prefPage.radioButton().isChecked(key);
+    }
+  });
+
+  it('Can select date format', () => {
+    /*
+    Select each option
+    Get values of options available and compare them to Regex
+    */
+    const dropBoxIndex = 2;
+
+    PagePo.goTo(endpoint);
+    prefPage.dropdownMenu().open(dropBoxIndex);
+    prefPage.listBox().isOpened();
+    prefPage.listBox().getListBoxItems().should('have.length', 5).then(($els) => {
+      const map = Cypress.$.map($els, el => el.innerText.trim());
+
+      for (const i in map) {
+        expect(map[i]).to.match(/([0-9]+(:[0-9]+)+)/)
+        prefPage.listBox().set(map[i]);
+        prefPage.listBox().isClosed();
+        prefPage.dropdownMenu().open(dropBoxIndex);
+        prefPage.listBox().isOpened();
+      }
+    });
+  });
+
+  it('Can select time format', () => {
```

Comment:
this can be validated by the text below the `Theme` button group

Reply:
Not sure what you mean here. We can discuss in meeting tomorrow.
In the mean time I've made updates to validate http request payload and response here: https://github.com/rancher/dashboard/pull/8705/commits/18cac9ace0c4c705828fe739c9279e059c8a9b85